articles/ctod.dd

```diff
@@ -906,16 +906,16 @@ $(H4 The C Way)
        C has problems with the DOS file system because a \ is an escape in a string.
        To specifiy the file c:\root\file.c you would use:
 $(CCODE
-char file[] = "c:\\root\\file.c";
+`char file[] = "c:\\root\\file.c";`
```

Comment:
This will make the font bold. That will result in some code blocks look different from others. Not sure if that matters.

Reply:
Once you pointed it out it bothered me enough that I went ahead and used ` ```c ` fences for these three items.